### ISSUE TITLE
Web UI can't display utf-8 characters in the log screen

### ISSUE THREAD

**Issue by truongdo** [rancher/rancher#13402]
**Rancher versions:**
rancher/rancher: 2.0
rancher/rancher-agent:

**Steps to Reproduce:**
Create a workload with a container keep printing out to stdout

**Results:**
Click "View logs" -> only a few messages are printed on the screen, then it stopped printing.
The log file under `/var/log/containers/` is fine.

**Comment by tfiduccia** [comment]
@truongdo - I am unable to reproduce your issue.  Can you give more details?

**Comment by truongdo** [comment]
@tfiduccia can we schedule a teamviewer session so that you can have access and ask for any information you need? cause I'm not sure what information I should give you to reproduce this problem.

**Comment by thaoula** [comment]
Hi @tfiduccia,

We are having the same issue.. the View Logs windows is just about always blank and displays disconnected in the right hand corner.

Regards,
Tarek

**Comment by tfiduccia** [comment]
@truongdo @thaoula - can you provide
1.  What type of cluster?
2.  Where is your rancher server?
3.  Is there a public image I can try?
4.  What type of workload & scale?

**Comment by thaoula** [comment]
Hi @tfiduccia,

1. Azure VM cluster (VM's created by Rancher)
2. Rancher Server is running on a VM in Azure
3. Mongo / Redis I think it happens to all.
4. Deployment, single pod.

I have seen the log displayed sometimes (vary rare) but mostly I get this -
![image](https://user-images.githubusercontent.com/349133/40087237-e550d338-58cb-11e8-8eef-104f0da0fa8a.png)

Thanks
Tarek

**Comment by truongdo** [comment]
@tfiduccia I managed to reproduce the problem with the below image.
The UI can't
`truongdq/test:latest`

Dockerfile
```
FROM ubuntu:latest
RUN echo "while true; do sleep 1; echo '12'; echo 'chào em'; done " > /test.sh && chmod +x /test.sh
CMD /test.sh
```

**Comment by truongdo** [comment]
I'm sorry, I hit the close button accidentally

**Comment by loganhz** [comment]
@truongdo This issue has been fixed in `v2.0.4`
